libs/net/kafka/KafkaListener`.cs

```diff
@@ -164,18 +165,25 @@ public async Task ConsumeAsync(Func<ConsumeResult<TKey, TValue>, Task<ConsumerAc
                     this.IsPaused = true;
                 }
             }
+            else
+            {
+                _logger.LogDebug("Retrying prior message from Kafka topic: '{topic}' key:'{key}'", _currentResult.Topic, _currentResult.Message.Key);
+            }
 
-            proceed = await action(_currentResult);
-            if (proceed == ConsumerAction.Proceed)
+            if (_currentResult != null)
```

Comment:
Somehow it was getting here with a null value.  I don't know how that is even possible and was unable to debug and find a way.